ltk/ltk.lisp

```diff
@@ -1154,23 +1146,19 @@ event to read and blocking is set to nil"
      finally (return result)))
   |#
 
-;; basic tk object
 (defclass tkobject ()
-  ((name :accessor name :initarg :name :initform nil)
-   )
+  ((name :accessor name :initarg :name :initform nil))
   (:documentation "Base class for every Tk object"))
 
-;; basic class for all widgets
```

Comment:
I removed the comment, as there is already a documentation string that says the same thing.